src/index.jsx

```diff
@@ -71,6 +72,21 @@ const App = () => (
       googlePlayUrl="https://play.google.com"
       handleAllTrackEvents={() => {}}
       socialLinks={socialLinks}
+      supportedLanguages={[
+        {
+          label: 'English',
+          value: 'en',
+        }, {
+          label: 'español',
+          value: 'es-419',
+        },
+      ]}
+      languageForm={{
+        activeLanguage: 'en',
+        screenReaderLabel: 'Choose Language',
```

Comment:
This and other text should be i18n'ed.

Reply:
Isn't this just an example of the passed in props (e.g. the marketing url is set to {{"https://www.example.com"}}?

Reply:
Ah, so it is!